resources/scripts/components/server/schedules/TaskDetailsModal.tsx

```diff
@@ -78,21 +79,27 @@ const TaskDetailsModal = ({ schedule, task }: Props) => {
 
     const submit = (values: Values, { setSubmitting }: FormikHelpers<Values>) => {
         clearFlashes('schedule:task');
-        createOrUpdateScheduleTask(uuid, schedule.id, task?.id, values)
-            .then(task => {
-                let tasks = schedule.tasks.map(t => t.id === task.id ? task : t);
-                if (!schedule.tasks.find(t => t.id === task.id)) {
-                    tasks = [ ...tasks, task ];
-                }
+        if (backupLimit === 0 && values.action === 'backup') {
+            console.error('This server is not allowed to create backups.');
```

Comment:
You don't need the `console.error` if you are showing a flash to the user.

Reply:
I was just following what was above it... Should it be removed in both places?

Reply:
Removed.